CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Headings should be (only add headings as needed):
 
 -  Fixed state and country question types shortcode output ([552](https://github.com/eventespresso/event-espresso-core/pull/552))
 -  Added CSS bulletproofing to protect registration form from worse-case scenarios ([542](https://github.com/eventespresso/event-espresso-core/pull/542))
+-  REST API "BETWEEN" operator ([560](https://github.com/eventespresso/event-espresso-core/pull/560))
```

Comment:
Some nitpicking here but I think its important, this changelog entry doesn't really follow the [suggested guidelines](https://github.com/eventespresso/event-espresso-core/blob/master/docs/A--Best-Practices/change-log.md):

- I think the description could be improved a bit to describe what exactly was fixed and why (while still keeping it short)
- It's not really a sentence.
- It's not declarative.

Typically for our pull request/issue titles (and for commit messages) I think its helpful if we use imperative language, but then in our changelog we use declarative.  So for example `Fix bug with x` would be the imperative version and then `Fixed bug with x` would be the declarative version.  If we follow that with crafting the pull request/issue titles, then it becomes much easier to do the changelog entry.

So if I was writing the changelog entry for this, I might put something like:

>  Fixed incorrect description of usage for the `BETWEEN` operator in the REST API documentation and also an issue where using the BETWEEN operator in the REST API was inconsistent.

Reply:
yeah good thing to point out, thanks. I also updated my original description as I think I came up with a better one-sentence description of the changes.